pump/storage/storage.go

```diff
@@ -135,7 +138,7 @@ func NewAppendWithResolver(dir string, options *Options, tiStore kv.Storage, tiL
 		return nil, errors.Trace(err)
 	}
 
-	writeCh := make(chan *request, chanSize)
+	writeCh := make(chan *request, options.KVChanCap)
```

Comment:
It seems like this is the input channel for `writeValueLog`, but the name `KVChanCap` suggests that it's for the output channel which is the input channel for `writeKV`.

Reply:
lol